### ISSUE TITLE
Multiple LAN segments - minor

### ISSUE THREAD

**Issue by mtompkins** [gclayburg/synology-diskstation-scripts#3]
First thanks for this handy script.

I have two of my multi-port 1813+ in use across different scopes. Segments are 192.168.0.0 & 192.168.1.0  I've manually enabled NAT across segments just for information - although not terribly relevant.

I have the reverse zone set correctly with the segment I want to populate dynamically (192.168.1.0), the script is also adding the 0.0 segment node's information as there is no filter to manage more than one segment (understandably).  It isn't a biggie - but just thought I'd mention it. 

FWIW Synology does a terrible job at isolating DHCP broadcasts across segments as well. I've had less than productive discussions with them and they've simple said "they are aware"...

**Comment by gclayburg** [comment]
Right, this script will update DNS from whatever DHCP leases are handed out.  There really isn't a filter mechanism at the moment that understands subnets.  If this functionality is important to you, perhaps you could create a pull request?

**Comment by mtompkins** [comment]
Thanks for the confirmation. I've really been hoping that Synology improves their approach to DHCP leases before I start to modify a few of the scripts I incorporate. (E.g. broadcast domains for DHCP are non-existent - an absolute nightmare when you use reservations)

I do promise that if I modify the script to handle multiple subnets I'll be sure to send a pull request.
